service/kvmconfig/v3/resource/configmap/delete.go

```diff
@@ -1,4 +1,4 @@
-package configmapv2
+package configmap
```

Comment:
Here it is the other way around so I guess only the diff is weird.

Reply:
Yes. Looks like files were the same.